### PR TITLE
Unit tests, examples, nonce3d [PRBUG-2115]

### PR DESCRIPTION
**Pull request by pretzelhands**

### CODE REVIEW

File: src/Mapper/RequestMapper.php [hunk 1]
```diff
@@ -167,6 +167,8 @@ public function mapSeamlessRequest(Transaction $transaction, $requestData)
             $requestData['token_id'] = $transaction->getTokenId();
         }
 
+        $requestData['wpp_options_cvv_hidden'] = true;
```
[the remark this answers is not in the document]
Comment (pretzelhands) [reply src/Mapper/RequestMapper.php]: Parameter is now only set for MyFavoritePayment transactions. This also eliminates the need for updating UI tests as they have not yet been provided for MyFavoritePayment.

File: src/Mapper/RequestMapper.php [hunk 2]
```diff
@@ -165,6 +165,7 @@ public function mapSeamlessRequest(Transaction $transaction, $requestData)
 
         if (null !== $transaction->getTokenId()) {
             $requestData['token_id'] = $transaction->getTokenId();
+            $requestData['wpp_options_cvv_hidden'] = true;
```
Comment (tomazpu) [inline src/Mapper/RequestMapper.php]: Please add a comment above the if why this option is added

File: src/Helper/RequestInspector.php [hunk 3]
```diff
@@ -65,13 +65,17 @@ public static function isValidRequest($request)
         }
```
Comment (ozada) [inline src/Helper/RequestInspector.php]: What was the reason in usage of static methods?

File: tests/acceptance/CreditCard3DSTokenizePurchaseErrorCards.feature [hunk 4]
```diff
@@ -5,7 +5,7 @@ Feature: CreditCard3DSTokenizePurchaseErrorCards
   Background:
     Given I am on "Create Credit Card UI Tokenize Page" page
 
-  @API-TEST @API-WDCEE-TEST @NOVA
+  # @API-TEST @API-WDCEE-TEST @NOVA
```
Comment (ozada) [inline tests/acceptance/CreditCard3DSTokenizePurchaseErrorCards.feature]: Why it's commented now?

File: examples/payments/CreditCard/menu.html [hunk 5]
```diff
@@ -14,7 +14,7 @@ <h3>Credit Card</h3>
         <a href="payments/CreditCard/cancel.html" class="list-group-item">Cancelling a transaction</a>
         <a href="payments/CreditCard/credit.html" class="list-group-item">Credit to a credit card</a>
         <a href="payments/CreditCard/pay-based-on-reserve.html" class="list-group-item">Payment after a reservation</a>
-        <a href="payments/CreditCard/pay-based-on-token.html" class="list-group-item">Recurring payment via token-id</a>
-        <a href="payments/CreditCard/pay-based-on-token-threed.html" class="list-group-item">Recurring payment via token-id with 3D parameters</a>
+<!--        <a href="payments/CreditCard/pay-based-on-token.html" class="list-group-item">Recurring payment via token-id</a>-->
```
[the remark this answers is not in the document]
Comment (tomazpu) [reply examples/payments/CreditCard/menu.html]: For this release the "one-click" example is hidden. That's the reason for comment and not delete.

File: src/Helper/RequestInspector.php (hunk 3 above)
Comment (tomazpu) [reply src/Helper/RequestInspector.php]: The usage of the method is only in tests, so I think it's ok.

File: tests/acceptance/CreditCard3DSTokenizePurchaseErrorCards.feature (hunk 4 above)
Comment (tomazpu) [reply tests/acceptance/CreditCard3DSTokenizePurchaseErrorCards.feature]: Same reason as above for the moment the one-click is commented out.

**Comment by ozada** [comment]
It's clear from my side. @tomazpu you can merge it to master.

**Pull request merged**